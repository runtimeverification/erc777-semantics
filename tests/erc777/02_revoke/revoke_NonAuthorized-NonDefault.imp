test.preamble();

// Testing revokeOperator(Operator) when Operator is not authorized but is a default operator.
// A token holder MAY revoke an already revoked operator. A RevokedOperator MUST be emitted each time.
// should pass

test.isOperatorFor(1, 7);
test.revokeOperator(1);
test.isOperatorFor(1, 7);
