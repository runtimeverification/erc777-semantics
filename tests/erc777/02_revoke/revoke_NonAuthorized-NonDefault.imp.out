1 is not operator for 7
7 is revoking operator 1
1 is not operator for 7
